examples/sample-layers/index.js

```diff
@@ -0,0 +1,3 @@
+export {default as EnhancedChoroplethLayer} from './enhanced-choropleth-layer';
+export {default as PlotLayer} from './plot-layer';
```

Comment:
Mainly a feeling but I think this is a strange mix of layers. They are so different in scope and ambition. I would have expected the demo layers to be their own stand alone examples (complete apps) that could be cloned, installed and run.